src/rust/engine/src/externs/options.rs

```diff
@@ -12,6 +11,8 @@ use options::{
 
 use std::collections::HashMap;
 
+pyo3::import_exception!(pants.option.errors, ParseError);
```

Comment:
We now raise our own ParseError instead of the generic ValueError. However it seemed like overkill to raise a bunch of ParseError subtypes (they were probably overkill to begin with even in the legacy parser).